### DISCUSSION TITLE
Next grammar update?

### DISCUSSION THREAD

**Discussion by jtbandes** [github-linguist/linguist#5197] [General]
I noticed the last update to all grammars was [v7.12.0](https://github.com/github/linguist/releases/tag/v7.12.0) back in November. I've made some updates to the [Swift grammar](https://github.com/textmate/swift.tmbundle) since then (with another important update forthcoming) and was just wondering if there's any plan for when the gem will be updated next? Thanks! 💖

**Comment by Nixinova** [comment]
Hopefully after my ungrouping PRs at least are added

**Comment by Nixinova** [comment]
Also, https://github.com/github/linguist/pull/5104#issuecomment-766673384 2w ago:

> I'm starting to get things in order for a new release.

**Comment by lildude** [comment]
Yes, the grammars will be updated when I make the next release which I'm aiming to do this week, provided I don't get steam-rolled by something more urgent.

**Reply by lildude** [reply]
Unfortunately, I've been delayed by more pressing work so the release will slip to next week - aiming for Monday but it could slip to Tuesday. I could deploy it tomorrow, but peeps get a little upset when their repo grammars etc change on a Friday 😬

**Comment by cssinate** [comment]
I'm really glad to hear this! The CSS language grammar was updated 17 days ago, and there's a change in there that fixes some broken stuff in SCSS highlighting. Thanks @lildude - I'm looking forward to it.

**Comment by lildude** [comment]
All done and now live on GitHub.com and Lightshow!! 🎉

**Reply by jtbandes** [reply]
Awesome! I see the changes live in some source files, but when I look at a rendered [Markdown file](https://github.com/apple/swift-evolution/blob/main/proposals/0296-async-await.md) with code blocks, it’s still seemingly using the old version. Does it just take some time to propagate to all files?

**Reply by lildude** [reply]
I'm not 100% sure how inline code blocks are handled but I suspect they are cached slightly differently. You can see the newly created gist I created using the same content at https://gist.github.com/lildude/a2ffd5e1ad0ec9060f9595c5a5c945d2 shows the updated syntax highlight so I'd expect the syntax to update the next time you make changes.

**Comment by codecat** [comment]
Are grammars updated with every Linguist release?

**Reply by lildude** [reply]
Yes. I'm aiming to do a release this week, but it's likely to slip to next week as I'm busier than I'd anticipated I'd be.

**Reply by codecat** [reply]
No worries, thanks for the heads up! 🥰
